pkg/fileutil/fileutil_test.go

```diff
@@ -34,8 +35,19 @@ func TestIsDirWriteable(t *testing.T) {
 	if err := os.Chmod(tmpdir, 0444); err != nil {
 		t.Fatalf("unexpected os.Chmod error: %v", err)
 	}
-	if err := IsDirWriteable(tmpdir); err == nil {
-		t.Fatalf("expected IsDirWriteable to error")
+	me, err := user.Current()
+	if err != nil {
```

Comment:
how about
if err != nil: t.Skipf("failed to get ..")
if me.Name == root, administrator: t.Skipf("running as superuser ..")